### ISSUE TITLE
file.line state crashes if _onlyif_ statement is used

### ISSUE THREAD

**Issue by crosscodr** [saltstack/salt#56347]
### Description of Issue
A file.line state crashes if _onlyif_ statement is used:

2020-03-10 18:49:17,107 [salt.utils.process:767 ][ERROR   ][21232] An un-handled exception from the multiprocessing process 'ProcessPayload' was caught:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/salt/utils/process.py", line 760, in wrapped_run_func
    return run_func()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3/dist-packages/salt/minion.py", line 1605, in _target
    run_func(minion_instance, opts, data)
  File "/usr/lib/python3/dist-packages/salt/minion.py", line 1600, in run_func
    return Minion._thread_return(minion_instance, opts, data)
  File "/usr/lib/python3/dist-packages/salt/minion.py", line 1787, in _thread_return
    timeout=minion_instance._return_retry_timer()
  File "/usr/lib/python3/dist-packages/salt/minion.py", line 2008, in _return_pub
    ret_val = self._send_req_sync(load, timeout=timeout)
  File "/usr/lib/python3/dist-packages/salt/minion.py", line 1418, in _send_req_sync
    with salt.transport.client.ReqChannel.factory(self.opts) as channel:
  File "/usr/lib/python3/dist-packages/salt/transport/client.py", line 25, in factory
    sync = SyncWrapper(AsyncReqChannel.factory, (opts,), kwargs)
  File "/usr/lib/python3/dist-packages/salt/utils/asynchronous.py", line 52, in __init__
    self.asynchronous = method(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/salt/transport/client.py", line 119, in factory
    import salt.transport.zeromq
  File "/usr/lib/python3/dist-packages/salt/__init__.py", line 10, in <module>
    import importlib
  File "/usr/lib/python3.6/importlib/__init__.py", line 31, in <module>
    sys.modules['importlib._bootstrap'] = _bootstrap
AttributeError: module 'sys' has no attribute 'modules'

### Setup
unlimit-apache2-coredumps:
  file.line:
    - name: /etc/systemd/system/multi-user.target.wants/apache2.service
    - mode: ensure
    - after: \[Service\]
    - content: LimitCORE=infinity
    - onlyif:
      - true

### Steps to Reproduce Issue
create file.line state with onlyif statement

### Versions Report
Salt Version:
           Salt: 3000
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 2.6.1
      docker-py: Not Installed
          gitdb: 2.0.3
      gitpython: 2.1.8
         Jinja2: 2.10
        libgit2: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.5.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 3.6.9 (default, Nov  7 2019, 10:44:02)
   python-gnupg: 0.4.1
         PyYAML: 3.12
          PyZMQ: 16.0.2
          smmap: 2.0.3
        timelib: Not Installed
        Tornado: 4.5.3
            ZMQ: 4.2.5
 
System Versions:
           dist: Ubuntu 18.04 bionic
         locale: UTF-8
        machine: x86_64
        release: 4.15.0-88-generic
         system: Linux
        version: Ubuntu 18.04 bionic

**Comment by max-arnold** [comment]
Duplicate of https://github.com/saltstack/salt/issues/56131

One possible workaround is to install `python3-pip`

**Comment by Ch3LL** [comment]
thanks for pointing that out @max-arnold 

closing as duplicate
